dbflute-engine/src/main/java/org/dbflute/friends/velocity/DfFlutistLog4JLogSystem.java

```diff
@@ -41,7 +42,7 @@ public DfFlutistLog4JLogSystem() {
     //                                                                 ===================
     @Override
     public void init(RuntimeServices rs) {
-        final String logfile = "./log/velocity.log";
+        final String logfile = DfBuildProperties.getInstance().getWorkDir() + "log/velocity.log";
```

Comment:
How about DfEngineWorkDir?
Many workDir variables can be trouble.
like this:
```
DfEngineWorkDir.toPath("log/velocity.log");
```
DfEngineWorkDir uses DfBuildProperties in it.

Reply:
add supplement:
```
String path = DfEngineWorkDir.toPath("log/velocity.log"); // ./log/velocity.log
```